src/HotChocolate/Core/src/Types.Scalars/ThrowHelper.cs

```diff
@@ -2,6 +2,28 @@ namespace HotChocolate.Types.Scalars
 {
     internal static class ThrowHelper
     {
+        public static SerializationException EmailAddressType_ParseLiteral_IsEmpty(IType type)
+        {
+            return new SerializationException(
+                ErrorBuilder.New()
+                    .SetMessage(ScalarResources.EmailAddressType_IsEmpty_ParseLiteral)
+                    .SetCode(ErrorCodes.Scalars.InvalidSyntaxFormat)
+                    .SetExtension("actualType", "EmailAddress")
```

Comment:
I have just realized something :D 
we do have the second string already as a constant
```csharp
        .SetExtension("actualType", WellKnownScalarTypes.EmailAddress)
```

Can you also make that change to the other methods in this file?